### PR TITLE
applications: nrf_desktop: Add user guide images

### PR DESCRIPTION
**Pull request by pdunaj**
Jira:DESK-900

Signed-off-by: Pawel Dunaj <pawel.dunaj@nordicsemi.no>

### CODE REVIEW

**Comment by pdunaj** [comment]
Hi @thst-nordic , do you know what can be this "Internal CI Error."?

**Comment by pdunaj** [comment]
Hi @greg-fer , I have added new images and rebased. Can you have a look. Unfortunately after the last changes in Zephyr I cannot render the documentation locally, again :(

File: applications/nrf_desktop/README.rst [hunk 1]
```diff
@@ -585,23 +603,111 @@ Otherwise, the old version is used.
...
+
+      .. figure:: /images/nrf_desktop_desktop_mouse_bottom.svg
+         :scale: 25 %
+         :alt: nRF Desktop desktop mouse (bottom view)
+
+         nRF Desktop desktop mouse (bottom view)
+
+   .. group-tab:: Keyboard
+
+      .. figure:: /images/nrf_desktop_keyboard_back.svg
```
Comment (greg-fer) [inline applications/nrf_desktop/README.rst:703]: **Image edit:**
Please remove the debug caption from this image.

File: applications/nrf_desktop/README.rst [hunk 2]
```diff
@@ -696,6 +808,13 @@ Hardware switch
 
 Precision Aim button
     * The button is located on the left side of the mouse, in the thumb area.
+
+      .. figure:: /images/nrf_desktop_gaming_mouse_top.svg
```
Comment (greg-fer) [inline applications/nrf_desktop/README.rst:812]: **Image edit:**
Please remove the debug caption from this image.

@pdunaj , do we need the mention of LED 1 on this image?
I think it should be OK to keep it, but without "- peer state", and with an explanation in the docs about this LED 1.

File: applications/nrf_desktop/README.rst [hunk 3]
```diff
@@ -708,25 +827,37 @@ Precision Aim button
...
++++++++++++++
 
 The following predefined buttons are assigned to peer control operations for the desktop mouse:
 
 Scroll wheel button
     * Long-press to initialize and confirm the peer erase.
       The scroll must be pressed before the mouse is powered up.
 
-        * |nRF_Desktop_confirmation_effect|
+      .. figure:: /images/nrf_desktop_desktop_mouse_side.svg
```
Comment (greg-fer) [inline applications/nrf_desktop/README.rst:838]: **Image edit:**
Please remove the debug caption from this image.

Also, @pdunaj, about middle button vs scroll wheel button. I think "scroll wheel button" is less confusing if the mouse has a button under the scroll. Some mice have DPI buttons located under the scroll wheel or above it, and they could also be considered "middle buttons".

File: applications/nrf_desktop/README.rst [hunk 4]
```diff
@@ -708,25 +827,37 @@ Precision Aim button
...
 Keyboard
-~~~~~~~~
+++++++++
 
 The following predefined buttons or button combinations are assigned to peer control operations for the keyboard:
 
 Page Down key
     * The Page Down key must be pressed while keeping the Fn modifier key pressed.
+
+      .. figure:: /images/nrf_desktop_keyboard_top.svg
```
Comment (greg-fer) [inline applications/nrf_desktop/README.rst:855]: @pdunaj , do we need LED1 on this image? I think we can keep it, but would have to add explanation in a bullet point below.

File: applications/nrf_desktop/README.rst [hunk 5]
```diff
@@ -741,35 +872,58 @@ Page Down key
...
 Dongle
-~~~~~~
+++++++
 
 The following predefined buttons are assigned to peer control operations for the dongle:
 
 SW1 button
+    * The SW1 button is located on the top of the dongle.
+
+      .. figure:: /images/nRF52840_dongle_front_callouts.svg
```
Comment (greg-fer) [inline applications/nrf_desktop/README.rst:882]: @pdunaj , same LED matter as for the keyboard and mouse. Let's keep the captions, but explain what the meaning of LEDs is.

File: applications/nrf_desktop/README.rst [hunk 6]
```diff
@@ -741,35 +872,58 @@ Page Down key
...
-Power Management
-================
+Each of the nRF Desktop hardware reference designs has a slot for a dedicated debug board.
+See the following figures for the exact location of these slots:
 
-Reducing power consumption is important for every battery-powered device.
+.. tabs::
 
-The nRF Desktop peripherals are either suspended or powered off when they are not in use for a predefined amount of time:
+   .. group-tab:: Gaming mouse
```
Comment (greg-fer) [inline applications/nrf_desktop/README.rst:902]: **Image edit:**
Please remove all captions _except_ the debug one from this image, and from the following two images.

**Comment by pdunaj** [comment]
Hi @greg-fer , please check if the images are ok. I have added few changes to text - please see if something needs modification.

**Comment by pdunaj** [comment]
I have split the change to appease the CI...

**Comment by pdunaj** [comment]
Splitting into multiple commits does not help the CI. We will need to move the images to the separate PRs. I will do it once we get review done.

**Comment by pdunaj** [comment]
@thst-nordic , @wiminordic , do you know if checkpatch will ever finish? this is the pr where check crashed because of the large number of chars.

**Comment by thst-nordic** [comment]
We have manually skipped the checkpatch, we can ignore it's status.

**Comment by pdunaj** [comment]
Hi @carlescufi , could you merge it without the checkpatch? It won't ever finish because of the svg images.

**Pull request merged**